examples/movie_view_ratings_local.py

```diff
@@ -0,0 +1,76 @@
+""" Demo of running PipelineDP locally, without any external data processing framework"""
+
+from absl import app
+from absl import flags
+import pipeline_dp
+
+from examples.example_utils import parse_file, write_to_file
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('input_file', None, 'The file with the movie view data')
+flags.DEFINE_string('output_file', None, 'Output file')
+
+
+def main(unused_argv):
+    # Here, we use a local backend for computations. Which does not depend on
+    # any pipeline framework and it is implemented in pure Python in
+    # PipelineDP. It keeps all data in memory and is not optimized for large data.
+    # For datasets smaller than ~tens of megabytes, local execution without any
+    # framework is faster than local mode with Beam or Spark.
+    backend = pipeline_dp.LocalBackend()
+
+    # Define the privacy budget available for our computation.
+    budget_accountant = pipeline_dp.NaiveBudgetAccountant(total_epsilon=1,
+                                                          total_delta=1e-6)
+
+    # Load and parse input data
+    movie_views = parse_file(FLAGS.input_file)
+
+    # Create a DPEngine instance.
+    dp_engine = pipeline_dp.DPEngine(budget_accountant, backend)
+
+    params = pipeline_dp.AggregateParams(
+        noise_kind=pipeline_dp.NoiseKind.LAPLACE,
+        metrics=[
+            # we can compute multiple metrics at once.
+            pipeline_dp.Metrics.COUNT,
+            pipeline_dp.Metrics.SUM,
+            pipeline_dp.Metrics.PRIVACY_ID_COUNT
+        ],
+        # Limit contributions of each user.
```

Comment:
We could also ay something more specific, e.g.:

....
# Maximum movies rated per user
max_partitions_contributed=2,

# At most one rating per user per movie
max_contributions_per_partition=1,

# Minimal movie rating 
min_value = 1,

# Maximal movie rating 
max_value = 5)